README.md

```diff
@@ -1,24 +1,244 @@
+```
+### Install
+
+Run this command: 
+
+Example command:
+
+```sh
+  cd blog-app
+  npm bundle install
+```
+
+To install dependencies and use rubocop
+
+### Usage
+
+Before we setup the database for the app make sure you check into "config/database.yml" to alter the file with your correct postgres credentials.
+
+Run this command to create the database and run all migrations: 
+
+```sh
+  rails db:create db:migrate
+```
+
+Now we are ready to run the app:
+
+```sh
+  rails server
+```
+<!--
+### Run tests
+
+Example command:
+
+```sh
+  cd spec
+rspec *spec.rb
+```
+-->
+
+### Deployment
+
+
+You can deploy this project using:
+
+Visual studio code or with the very own ruby ird
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Eduardo Villarreal**
+
+- GitHub: [@evillaz](https://github.com/evillaz)
+- Twitter: [@evillazz](https://twitter.com/evillazz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/eduardo-villarreal-144a8925a)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Setup and controllers.]**
+- **[Controllers specs.]**
+- **[Views.]**
+- **[Forms.]**
+- **[Integration specs for Views and fixing n+1 problems.]**
+- **[Add authorization rules.]**
+- **[Add API endpoints.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project  don't doubt on make any comments or try to reach me out.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<!-- Give credit to everyone who inspired your codebase. -->
+
+I would like to thank microverse for the opportunity on working each other as colleagues and the learning opportunities it brings us.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional)
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Did you find hard to create this project?**
+
+  - I tried my best and follow the guidelines to create this project. I got hard times on some of the workflow but i think i just have to keep learning everyday and give my best try every time.
+
+- **What was your thinking process to solve the requirements?**
+
+  - First i tried to address each one of the requirements and solve them individually to keep integrating them as i was progressing.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please kindly ensure that the license file is linked appropraitely, I got 404 error. Implemeting this changes will make your repo to be more professional. 👍